src/assets/styles/header.scss

```diff
@@ -0,0 +1,279 @@
+.header {
+  background: $dark;
+  text-align: center;
+  padding: 1em;
+}
+
+.header-inner {
+  margin: 0 auto;
+  width: 80%;
+  max-width: 500px;
+}
+
+.toolbar {
+  position: relative;
+}
+
+.toolbar.-desktop {
+  display: none;
+}
+
+.logo {
+  max-width: 210px;
+}
+
+.cta {
+  font-family: "Freight Sans CMP";
+  color: $white;
+  font-size: 2em;
+  margin: .1em 0;
+}
+
+.search-wrap {
+  height: 2em;
+  overflow: auto;
```

Comment:
Just remembered I used this poor-man's clearfix in a couple places....I might update this